recipes/minizip-ng/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+import os
+from conans import ConanFile, tools, CMake
+
+
+class MinizipNgConan(ConanFile):
+    name = "minizip-ng"
+    description = "Fork of the popular zip manipulation library found in the zlib distribution."
+    topics = ("conan", "compression", "zip")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/zlib-ng/minizip-ng"
+    license = "Zlib"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake", "cmake_find_package", "pkg_config"
+    settings = "os", "compiler", "build_type", "arch"
+    provides = "minizip"
```

Comment:
```suggestion
```

conditionally set below in `configure()`

Reply:
done